docs/data/base/components/snackbar/UnstyledSnackbarIntroduction/tailwind/index.tsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import { Transition } from 'react-transition-group';
+import { useTheme } from '@mui/system';
+import CheckRoundedIcon from '@mui/icons-material/CheckRounded';
+import CloseIcon from '@mui/icons-material/Close';
+import { Snackbar } from '@mui/base/Snackbar';
+import { SnackbarCloseReason } from '@mui/base/useSnackbar';
+
+function useIsDarkMode() {
+  const theme = useTheme();
+  return theme.palette.mode === 'dark';
+}
+
+export default function UnstyledSnackbarIntroduction() {
+  const isDarkMode = useIsDarkMode();
+
+  const [open, setOpen] = React.useState(false);
+  const [exited, setExited] = React.useState(true);
+  const nodeRef = React.useRef(null);
+
+  const handleClose = (_: any, reason: SnackbarCloseReason) => {
+    if (reason === 'clickaway') {
+      return;
+    }
+
+    setOpen(false);
+  };
+
+  const handleClick = () => {
+    setOpen(true);
+  };
+
+  const handleOnEnter = () => {
+    setExited(false);
+  };
+
+  const handleOnExited = () => {
+    setExited(true);
+  };
+
+  return (
+    <div className={`${isDarkMode ? 'dark' : ''}`}>
+      <button
```

Comment:
@zanivan I am not sure about the Tailwind CSS buttons styles, they are too different than the others. Should we update the demos?

Reply:
In this case, all the three styles are too different from the buttons elsewhere. They indeed look great, and similar to the buttons on marketing pages, but I believe we could simplify their design a little and make them similar to what we have on [Button demos](https://mui.com/base-ui/react-button/). I went ahead and made this change, what do you think?

Reply:
Looks great, we can apply these styles on all demos in a dedicated PR 👍